src/wta/activity_transitions.py

```diff
@@ -90,11 +90,37 @@ def __multiprocess_run(log, log_ids, calendar, parallel_activities):
 
     for h in tqdm(handles, desc='Waiting for tasks to finish'):
         done = h.done()
-        result = h.result()
-        if done and not result.empty:
-            all_items.append(result)
-
-    return all_items
+        if done:
+            result_transitions = h.result()
+            if not result_transitions.empty:
+                all_transitions.append(result_transitions)
+
+    # Concatenate the DataFrames vertically into a single DataFrame
+    concatenated_transitions = pd.concat(all_transitions, ignore_index=True)
+    concatenated_transitions = convert_time_columns_to_seconds(concatenated_transitions)
+    try:
+        ordered_columns = [
+            'start_time',
+            'end_time',
+            'source_activity',
+            'source_resource',
+            'destination_activity',
+            'destination_resource',
+            'case_id',
+            'wt_total',
+            'wt_contention',
+            'wt_batching',
+            'wt_prioritization',
+            'wt_unavailability',
+            'wt_extraneous'
+        ]
+
+        concatenated_transitions = concatenated_transitions[ordered_columns]
+
+    except KeyError as e:
+        print(f"Error: {e}.")
+
+    return concatenated_transitions
 
```

Comment:
There are several issues with this change:

- You're taking the function that's responsible only for running another function in a parallel way and adding business logic into it too. It's better to encapsulate such kind of post-processing into a standalone function and apply after this `__multiprocess_run` function returns.
- Also, you're repeating the code for sequential and parallel runs but you post-process results slightly differently. Is it intentional?
- There's a clearer way of checking if a column is present in the data frame, https://stackoverflow.com/questions/24870306/how-to-check-if-a-column-exists-in-pandas. You don't have to raise and catch the exception.